docs/config/configuring-authentication.rst

```diff
@@ -168,9 +168,9 @@ Import the client certificate into the ``Keychain`` and then refer to it by its
 Linux
 -----
 
-.. code-block:: console
+.. prompt:: bash $
 
```

Comment:
(as discussed) - auto is used when you don't want every line in the code sample to have the same prompt. In this example, we want the first line to have the $ prompt, but not the second line, as it is a result of the command. 

```bash $ auto``` will create a prompt of $ where it finds one in the text, otherwise no prompt is included